PXSDK/Classes/PXUser.m

```diff
@@ -0,0 +1,25 @@
+//
+//  PXUser.m
+//  PXSDK
+//
+
+#import "PXUser.h"
+
+@implementation PXUser
+
+- (id)initWithDictonary:(NSDictionary *)userDict {
+    self = [super init];
+    if (self) {
```

Comment:
maybe it's easier to make mapping dictionary e.g.
{ @"timeForIAPOffer" : @"params1", ... }
and set values using KVC?